### DISCUSSION TITLE
Can I generate the QR directly with Web3Modal?

### DISCUSSION THREAD

**Discussion by ssh07240724** [WalletConnect/walletconnect-monorepo#3943] [Developer Support]
I want to display the WalletConnect QR via Web3Modal(ReactNative, Javascript library) without wallet selection step.
Is it possible to skip the second step and create a QR?

Thank you.

<img width="1296" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/17022474/b23a5d56-fd73-4cdb-9580-f5860413344c">

**Comment by glitch-txs** [comment]
Hi, I'm not sure if it's possible, but could you try this

```ts
const { open } = useWeb3Modal()

open({view: 'ConnectingWalletConnect'})
```
